swagger.yaml

```diff
@@ -221,6 +221,60 @@ paths:
           description: Wrong content type (should be plain text)
         422:
           description: Missing parameters
+  /projects/sharing/{token}:
+    parameters:
+     - in: path
+       name: token 
+       schema:
+         type: string 
+       required: true
+    post:
+      tags:
+        - projects
+        - collaboration
+      summary: Adds user to a collaboration list for a particular project
+      security:
+        - user_id: []
+      description: Adds users to a collaboration list if the token is not expired
+      responses:
+        201:
+          description: Successfully added user to a collaboration list
+        400:
+          description: Malformed Request
+        401:
+          description: Unauthorized
```

Comment:
How can this happen?

Reply:
Also missing 403 and 422 probably

Reply:
401 is in case no X-User-Id header is provided, i'll add 422 though. No 403 is fine, as this endpoint gives access to a resource.